src/engine/SCons/Tool/MSCommon/vc.py

```diff
@@ -80,25 +80,76 @@ class BatchFileExecutionError(VisualCException):
     "i486"      : "x86",
     "i586"      : "x86",
     "i686"      : "x86",
-    "ia64"      : "ia64",
-    "itanium"   : "ia64",
+    "ia64"      : "ia64",      # deprecated
+    "itanium"   : "ia64",      # deprecated
     "x86"       : "x86",
     "x86_64"    : "amd64",
-    "x86_amd64" : "x86_amd64", # Cross compile to 64 bit from 32bits
+    "arm"       : "arm",
+    "arm64"     : "arm64",
+    "aarch64"   : "arm64",
+}
+ 
+# get path to the cl.exe dir for newer VS versions
+# based off a tuple of (host, target) platforms
+_HOST_TRGT_TO_CL_DIR_GREATER_THAN_14 = {
```

Comment:
change TRGT to TARGET please (also below)

Reply:
updated in f47c258